<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubeDialog</class>
 <widget class="QDialog" name="CubeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="coord">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>原点</string>
   </property>
  </widget>
  <widget class="QLabel" name="cubesize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>长宽高</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="coordX">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coordY">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="coordZ">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cubesizeZ">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cubesizeX">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cubesizeY">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
